src/components/Leaderboard.tsx

```diff
@@ -94,21 +38,49 @@ const Leaderboard: React.FC<IProps> = ({ content, limit = 100 }) => (
           emoji = ":3rd_place_medal:"
         }
         return (
-          <Item key={idx} to={hasGitHub ? `${githubUrl}${username}` : "#"}>
-            <ItemNumber>{idx + 1}</ItemNumber>
-            <Avatar src={avatarImg} alt={avatarAlt} />
-            <TextContainer>
-              <ItemTitle>{name}</ItemTitle>
-              <ItemDesc>
+          <Link
+            key={idx}
+            textDecor="none"
+            display="flex"
+            justifyContent="space-between"
+            alignItems="center"
+            color="text"
+            boxShadow="0 1px 1px tableItemBoxShadow"
+            mb={0.25}
+            p={4}
+            w="100%"
+            _hover={{
+              textDecor: "none",
+              borderRadius: 0.5,
+              boxShadow: "0 0 1px primary",
+              background: "tableBackgroundHover",
+            }}
+            to={hasGitHub ? `${githubUrl}${username}` : "#"}
+          >
+            <Box mr={4} opacity="0.4">
+              {idx + 1}
+            </Box>
+            <Avatar
+              src={avatarImg}
+              name={avatarAlt}
+              mr={4}
+              h={10}
+              w={10}
+              borderRadius="50%"
```

Comment:
is this necessary with the Chakra Avatar component? isn't that component already using this radius by default?